src/main/java/org/zalando/nakadi/domain/EventTypeBase.java

```diff
@@ -224,4 +230,13 @@ public void setCompatibilityMode(final CompatibilityMode compatibilityMode) {
     public void setValidationStrategies(final List<ValidationStrategyConfiguration> validationStrategies) {
         this.validationStrategies = validationStrategies;
     }
+
+    @Nullable
```

Comment:
Yes, I don't want to show users this null. 
I think it's better to hide property than to show null to users.